Libraries/pbxbuild/Sources/Phase/ProductTypeResolver.cpp

```diff
@@ -335,6 +465,15 @@ resolve(Phase::Environment const &phaseEnvironment, Phase::Context *phaseContext
     }
 
     /*
+     * Create the product's module maps, if required.
+     */
+    if (pbxsetting::Type::ParseBoolean(environment.resolve("DEFINES_MODULE"))) {
```

Comment:
http://clang.llvm.org/docs/Modules.html#module-maps has some documentation -- but even after reading it several times, I still find module maps to be pretty confusing 😅